src/lib.rs

```diff
@@ -1219,6 +1307,70 @@ pub fn handle_compile_subcommand(
     Ok(json!({"descriptor": descriptor.to_string()}))
 }
 
+/// Proof of reserves verification sub-command
+///
+/// Proof of reserves options are described in [`CliSubCommand::ExternalReserves`].
+#[cfg(all(feature = "reserves", feature = "electrum"))]
+pub fn handle_ext_reserves_subcommand(
+    network: Network,
+    message: String,
+    addresses: Vec<String>,
+    psbt: String,
+    electrum_opts: ElectrumOpts,
+) -> Result<serde_json::Value, Error> {
+    let psbt = base64::decode(&psbt).unwrap();
+    let psbt: PartiallySignedTransaction = deserialize(&psbt).unwrap();
+    let client = Client::new(&electrum_opts.server).unwrap();
```

Comment:
Although there are many places where we are using `unwrap`s currently in the lib, the goal is to handle them properly eventually. In that spirit it would be helpful if you can handle these unwraps. If there is no auto error conversion available for these functions, you can use the `Error::Generic()` variant to map them.